metadata-ingestion/tests/integration/kafka-connect/test_kafka_connect.py

```diff
@@ -338,7 +338,7 @@ def loaded_kafka_connect(kafka_connect_runner):
 
     # Give time for connectors to process the table data
     kafka_connect_runner.wait_until_responsive(
-        timeout=30,
+        timeout=40,
```

Comment:
is this needed ?

Reply:
As I added snowflake sink connector installation command, I had to increase this timeout otherwise execution of test cases were giving issues. If we remove that command, I will update it back to 30.